virt/live_migration/virt-migrating-vm-on-secondary-network.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY 
+[id="virt-migrating-vm-on-secondary-network"]
+= Migrating a virtual machine over a dedicated secondary network
+include::modules/virt-document-attributes.adoc[]
+:context: virt-migrating-vm-on-secondary-network
+
+toc::[]
+
+You can configure a dedicated xref:../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[secondary Multus network] for live migration. A dedicated network minimizes disruption to tenant workloads due to network saturation when virtual machine live migration is triggered.
+
+
+include::modules/virt-configuring-secondary-network-vm-live-migration.adoc[leveloffset=+1]
```

Comment:
What would you think of adding an additional resources section with an xref to the page that discusses the `liveMigrationConfig` stanza? It is probably not 100% necessary, but it occurred to me as a suggestion.

Reply:
Wouldn't hurt to provide the extra resource. That would be the [Live migration limits and timeouts](https://docs.openshift.com/container-platform/4.9/virt/live_migration/virt-live-migration-limits.html) page I believe.